.github/workflows/go.yml

```diff
@@ -0,0 +1,43 @@
+name: Go
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: fmt
```

Comment:
Is there any need for this task when ```make all``` performs 'fmt' anyway?

Reply:
good point, thanks. It makes sense to remove it